### PR TITLE
(SIMP-6263) Revert to using camptocamp/systemd

### PR DESCRIPTION
**Pull request by sharkbruhaha**
https://gitlab.com/simp/simp-core/pipelines/53770999

### CODE REVIEW

File: Puppetfile.stable [hunk 1]
```diff
@@ -41,14 +41,9 @@ mod 'camptocamp-kmod',
   :git => 'https://github.com/simp/puppet-kmod.git',
   :tag => '2.2.0'
 
-# We are maintaining a simp-master branch of this module
-# because we needed changes earlier, and, due to a simp-adapter
-# bug WRT RPM upgrades with name changes, can't revert back
-# to camptocamp/systemd.  We will revert when we redesign module
-# RPM packaging (SIMP-6.4.0 or later).
 mod 'simp-systemd',
```
Comment (lnemsick-simp) [inline Puppetfile.stable]: please change from simp-systemd to camptocamp-systemd for clarity.  In Puppetfiles we list <owner>-<module name>

File: Puppetfile.tracking [hunk 2]
```diff
@@ -41,14 +41,9 @@ mod 'camptocamp-kmod',
   :git => 'https://github.com/simp/puppet-kmod.git',
   :tag => '2.3.0'
 
-# We are maintaining a simp-master branch of this module
-# because we needed changes earlier, and, due to a simp-adapter
-# bug WRT RPM upgrades with name changes, can't revert back
-# to camptocamp/systemd.  We will revert when we redesign module
-# RPM packaging (SIMP-6.4.0 or later).
 mod 'simp-systemd',
   :git => 'https://github.com/simp/puppet-systemd.git',
```
Comment (lnemsick-simp) [inline Puppetfile.tracking]: camptocamp-systemd

File: metadata.json [hunk 3]
```diff
@@ -24,7 +24,7 @@
     },
     {
       "name": "simp/systemd",
```
Comment (lnemsick-simp) [inline metadata.json]: camptocamp/systemd

File: src/assets/simp/build/simp.spec [hunk 4]
```diff
@@ -23,7 +23,7 @@ Obsoletes: pupmod-simp-mcollective <= 3.0.0
 
 # Core SIMP Requirements
 Requires: pupmod-camptocamp-kmod >= 2.3.0, pupmod-camptocamp-kmod < 3.0.0
-Requires: pupmod-simp-systemd >= 2.1.0, pupmod-simp-systemd < 3.0.0
+Requires: pupmod-simp-systemd >= 2.2.0, pupmod-simp-systemd < 3.0.0
```
Comment (lnemsick-simp) [inline src/assets/simp/build/simp.spec]: pupmod-camptocamp-systemd

File: src/assets/simp/build/simp.spec [hunk 5]
```diff
@@ -245,6 +245,9 @@ fi
 # Post uninstall stuff
 
 %changelog
+* Tue Mar 26 2019 Joseph Sharkey <shark.bruhaha@gmail.com> - 6.4.0-0
+- Updated systemd package version
```
Comment (lnemsick-simp) [inline src/assets/simp/build/simp.spec]: Please add comment about change in ownership.  See other changelog entries.

File: src/assets/simp/build/simp.spec (hunk 4 above)
Comment (sharkbruhaha) [reply src/assets/simp/build/simp.spec]: should this actually just be changed to `puppet-systemd` ? Since that is what the repo is called?

Comment (lnemsick-simp) [reply src/assets/simp/build/simp.spec]: The name of the RPM will be constructed from the top-level 'name' key in the metadata.json

Comment (sharkbruhaha) [reply src/assets/simp/build/simp.spec]: ahhh gotcha, thanks!

Comment (sharkbruhaha) [reply src/assets/simp/build/simp.spec]: then should be just `camptocamp-systemd`

Comment (lnemsick-simp) [reply src/assets/simp/build/simp.spec]: No, it is constructed from name.  pupmod-<name>

Comment (sharkbruhaha) [reply src/assets/simp/build/simp.spec]: mmm, gotcha

**Comment by sharkbruhaha** [comment]
ISO successfully builds

**Comment by lnemsick-simp** [comment]
I've pushed a change to update  the version of mate-themes for el7 builds.  (Version available in EPEL was updated). The change to the PR made it into sharkbruhaha's repo, but is not showing here yet.  With that change, both el6 and el7 SIMP ISO builds were successful.

When the change shows up here, I'll approve.

**Pull request merged**